user-agent-rewriter/popup/choose_ua.css

```diff
@@ -1,14 +1,19 @@
-html, body {
+html, body, .ua-choices {
   height: 100px;
   width: 100px;
```

Comment:
The default font on my system (manjaro Linux) resolves to ` "DejaVu Serif"`. It's a bit too wide:
![](http://imgur.com/IPd73a5)
102px seems to be the limit.

However, the popup works just fine with no `width` rule.